ios/REANodesManager.m

```diff
@@ -382,7 +428,13 @@ - (void)dispatchEvent:(id<RCTEvent>)event
   event.eventName];
 
   if (_eventHandler != nil) {
-    _eventHandler(eventHash, event);
+    __strong REAEventHandler eventHandler = _eventHandler;
```

Comment:
this should not be needed. The block below already captures `self`, so you can access event handler using `self->_eventHandler`